packages/block-library/src/home-link/index.php

```diff
@@ -129,7 +129,7 @@ function render_block_core_home_link( $attributes, $content, $block ) {
 
 	$aria_current = is_home() || ( is_front_page() && 'page' === get_option( 'show_on_front' ) ) ? ' aria-current="page"' : '';
 
-	$html = '<li ' . $wrapper_attributes . '><a class="wp-block-home-link__content wp-block-navigation-item__content" rel="home"' . $aria_current;
+	$html = '<li ' . $wrapper_attributes . '><a class="wp-block-home-link__content wp-block-navigation-item__content" rel="home"' . esc_attr( $aria_current );
```

Comment:
`aria_current` is not user input but defined above. Also `esc_attr` is for the attribute values, no? This produces duplicate `"` now.

Reply:
> aria_current is not user input but defined above

Understood, but the best practice in WP Core is to escape at the point of _output_. This reduces the risk of missing escaping data.

For example, whilst the aria is defined above now, one day that could change and there could be a different method of computing that value. Therefore if we escape when building the HTML we reduce the risk of regressions in future.

> Also esc_attr is for the attribute values, no? This produces duplicate " now.

Sorry my bad. I will have to escape where the value is being built, or change the way the markup is built. Echoing values directly into HTML string without escaping is probably not ideal though...

Reply:
I'm going to leave this unescaped and hard coded for now only because the aria is very unlikely to change and it's hardcoded at the point of output.